Add release note for new recover-broker-info command (#630)
submariner-io/subctl: release-notes/20230330-recover-broker-info.md

<!-- markdownlint-disable MD041 -->

> A new `subctl recover-broker-info` command is added that recovers a lost `broker-info.subm` file.

Reviewer: ```suggestion
Added a `subctl recover-broker-info` command to recover lost `broker-info.subm` files.
```

Reviewer: (this is passive voice, see https://www.hamilton.edu/academics/centers/writing/seven-sins-of-writing/1)

Author: there is just 1 `broker-info.subm` file that is recovered. So "file". Changed the sentence to passive voice.
